README.md

```diff
@@ -23,7 +23,7 @@ differently with an easy to use yet very powerful syntax.
 
 ## Usage
 
-Create an instance based on an existing `RestTemplate`:
+Create an instance based on an existing `RestTemplate` (also see section *RestTemplate Configuration*):
```

Comment:
You can use a real link:

    [*RestTemplate Configuration*](#resttemplate-configuration)